### DISCUSSION TITLE
Case Insensitive Regex

### DISCUSSION THREAD

**Discussion by txtyash** [hyprwm/Hyprland#2810] [Q&A] [answered]
What could be a better window rule for browser tabs with youtube? Right now I'm using this:
```
title:.*(youtube|YouTube|Youtube).*
```
How can I use case insensitive flag to make this cleaner?

**Comment by Xenorf** [comment, marked as answer]
Hey there, theoretically this should work:
`.*(?i)youtube.*` 
In practice this flag doesn't seem to work, maybe an issue on Hyprland. My workaround is the following regex, at least it matches every possibilities:
`.*[Yy][Oo][Uu][Tt][Uu][Bb][Ee].*`
